syntax = "proto3";

package chat;

service ChatService {
  rpc SendMessage (stream ChatMessage) returns (stream ChatMessage);
  rpc SendMessageSingle (ChatMessage) returns (ChatMessage);
}

message ChatMessage {
  string name = 1;
  string message = 2;
}